lib/uksched/include/uk/sched.h

```diff
@@ -212,6 +217,7 @@ struct uk_thread *uk_sched_thread_create_fn1(struct uk_sched *s,
 					     uk_thread_fn1_t fn1,
 					     void *argp,
 					     size_t stack_len,
+					     size_t auxstack_len,
```

Comment:
Since the public functions of `uk_thread` are changed, please make sure no regressions are introduced in external libs. Example from `lib-lwip`:
```
/home/mpp/devel/unikraft/uk_vfork/app-elfloader/.unikraft/libs/lwip/threads.c: In function ‘sys_thread_new’:
/home/mpp/devel/unikraft/uk_vfork/app-elfloader/.unikraft/libs/lwip/threads.c:54:13: error: too few arguments to function ‘uk_sched_thread_create_fn1’
   54 |         t = uk_sched_thread_create_fn1(s,
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/mpp/devel/unikraft/uk_vfork/app-elfloader/.unikraft/libs/lwip/threads.c:33:
/home/mpp/devel/unikraft/uk_vfork/app-elfloader/.unikraft/unikraft/lib/uksched/include/uk/sched.h:216:19: note: declared here
  216 | struct uk_thread *uk_sched_thread_create_fn1(struct uk_sched *s,
      |                   ^~~~~~~~~~~~~~~~~~~~~~~~~~
  CC      liblwip: ip.o
```

Reply:
The diff is simple, indeed, I had the modification locally and forgot about it. Fix is as simple as
```diff
diff --git a/threads.c b/threads.c
index ea7f008..f431256 100644
--- a/threads.c
+++ b/threads.c
@@ -53,7 +53,7 @@ sys_thread_t sys_thread_new(const char *name, lwip_thread_fn thread, void *arg,
 
     t = uk_sched_thread_create_fn1(s,
                        (uk_thread_fn1_t) thread, arg,
-                       (size_t) stacksize,
+                       (size_t) stacksize, 0,
                        false,
                        false,
                        name,
```


When we get close to merging this stuff I will submit a PR for this :+1: